File: src/main/scala/com/temportalist/thaumicexp/common/tile/IOperator.scala
package com.temportalist.thaumicexp.common.tile

import net.minecraft.item.ItemStack

/**
 *
 *
 * @author TheTemportalist
 */
trait IOperator {

	def getInput(): ItemStack

	def getOutput(): ItemStack

	def finishedOperation(setInput: ItemStack, toOutput: ItemStack, data: Array[Any]): Unit

}

File: src/old/com/temportalist/thaumicexpansion/common/lib/IOperator.java
package com.temportalist.thaumicexpansion.common.lib;

import net.minecraft.item.ItemStack;

/**
 * @author TheTemportalist
 */
public interface IOperator {

	public ItemStack getInput();

	public ItemStack getOutput();

	public void finishedOperation(ItemStack setInput, ItemStack toOutput);

}
